crates/chia-sdk-types/src/run_puzzle.rs

```diff
@@ -0,0 +1,19 @@
+use clvmr::{
+    reduction::{EvalErr, Reduction},
+    Allocator, NodePtr,
+};
+
+pub fn run_puzzle(
+    allocator: &mut Allocator,
+    puzzle: NodePtr,
+    solution: NodePtr,
+) -> Result<NodePtr, EvalErr> {
+    let Reduction(_cost, output) = clvmr::run_program(
+        allocator,
+        &clvmr::ChiaDialect::new(0),
+        puzzle,
+        solution,
+        11_000_000_000,
```

Comment:
much better than u64::MAX

Reply:
It's still way too high since I forgot this doesn't take into account conditions or puzzle reveal and solution size in bytes multiplied by 12,000. But I don't have a better idea at the moment